release.yaml

```diff
@@ -47,16 +49,16 @@ rancher-monitoring:
   - 103.0.1+up45.31.1
```

Comment:
Please, keep only the `rancher-monitoring` version that you're currently working on and the one that should be released in the upcoming _ Rancher v2.8.3_ Release.

Reply:
@lucasmlp  this is not going in the 2.8.3 release